benchmark/_http-benchmarkers.js

```diff
@@ -112,7 +112,7 @@ exports.run = function(options, callback) {
   child.stderr.pipe(process.stderr);
 
   let stdout = '';
-  child.stdout.on('data', (chunk) => stdout += chunk.toString());
+  child.stdout.on('data', (chunk) => { return stdout += chunk.toString(); });
```

Comment:
Even after realizing https://github.com/nodejs/node/pull/10511#issuecomment-269651342, in this case, the `{}` being mandatory is arguable, I no longer have a opinion on that, but the `return` should *not* be present, event listeners should not return values.

I assume this change was done mechanically by eslint, because it is a literal translation of what the code did before, but isn't what was intended by the original author, it was just an unneeded side-effect.